ui/desktop/app/routes/scopes/scope/projects/settings/index.js

```diff
@@ -4,5 +4,39 @@
  */
 
 import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
 
-export default class ScopesScopeProjectsSettingsIndexRoute extends Route {}
+export default class ScopesScopeProjectsSettingsIndexRoute extends Route {
+  @service ipc;
+
+  async model() {
+    let cacheDaemonStatus,
+      formattedCacheVersion,
+      formattedCliVersion,
+      formattedDesktopVersion;
+    const { versionNumber: cliVersion } =
+      await this.ipc.invoke('getCliVersion');
+    formattedCliVersion = `v${cliVersion}`;
+
+    const { desktopVersion } = await this.ipc.invoke('getDesktopVersion');
+    formattedDesktopVersion = `v${desktopVersion}`;
+    try {
+      cacheDaemonStatus = await this.ipc.invoke('cacheDaemonStatus');
+    } catch (e) {
+      __electronLog?.warn('cache daemon is not running', e.message);
```

Comment:
I don't think this is strictly needed since we already have a log statement if the status fails in electron [here](https://github.com/hashicorp/boundary-ui/blob/10c128ee3baf695e350679f82122ef0cfb8cdd8a/ui/desktop/electron-app/src/services/cache-daemon-manager.js#L40)